runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -516,6 +525,12 @@ private void killedOne()
 		infoTimer = Instant.now();
 	}
 
+	private boolean doubleTroubleExtraKill()
+	{
+		return WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID() == GROTESQUE_GUARDIANS_REGION &&
```

Comment:
This Varp is visible only with slayer rewards screen open, so you should probably save it somewhere first time it is non 0

Reply:
Its always set for me 🤔